src/lib/components/molecules/party-profile/style.module.css

```diff
@@ -0,0 +1,74 @@
+.text {
+  font-family: var(--text-sans);
+  font-size: var(--sans-small);
+  line-height: var(--sans-line-height);
+  font-weight: 700;
+  margin-left: var(--space-1);
+}
+
+.container {
+  font-weight: 700;
+  min-width: 290px;
+  max-width: 530px;
+  margin-right: 6px;
+}
+
+.title {
+  font-family: var(--text-headline);
+  font-size: var(--headline-xsmall);
+  color: var(--primary-text-color);
+  font-weight: 500;
+  line-height: 1.1;
+
+  &:before {
+    content: '';
+    height: 4px;
+    width: 2.2em;
+    border-radius: 50px;
+    display: block;
+  }
+
+}
+
+.subtitle {
+  font-family: var(--text-headline);
+  font-size: var(--headline-xsmall);
+  color: var(--primary-text-color);
+  font-style: italic;
+  font-weight: 500;
+  line-height: 1.3;
+
+}
+
+.blurb {
+  font-family: var(--text-serif);
```

Comment:
Also here - can you add `color: var(--primary-text-color);` currently it's not assigned a color here so just using defaults, making it behave differently from the title 
<img width="346" alt="Screenshot 2024-03-20 at 14 28 31" src="https://github.com/guardian/interactive-component-library/assets/10324129/a17add51-0335-4e72-891e-08f1b5a03d83">